Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -53,7 +53,9 @@ static bool CompressCB(const std::string&, float, void*);
 
 GameList::GameList(QWidget* parent) : QStackedWidget(parent)
 {
-  m_model = Settings::Instance().GetGameListModel();
+  m_model = new GameListModel(this);
```

Comment:
`m_model` should also use `shared_ptr` to ensure GameListModel is not deleted at least until GameList itself is deleted.

Reply:
As mentioned above, QPointer has been used instead, so that the GameListModel ownership is then well defined by the GameList.